docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,391 @@
+
+| Values |
+| --- |
+| ON |
+| OFF |
+| FORCE |
+| FORCE_PLUS_PERMANENT |
+
+### audit_log_filter_buffer_size
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-buffer-size |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 1048576  |
+| Minimum value | 4096 |
+| Maximum value | 18446744073709547520 |
+| Units | byes |
+| Block size | 4096 |
+
+Defines the buffer size in multiples of 4096 when logging is asynchronous. The contents for events are stored in a buffer. The contents are stored until the contents are written.
+
+The plugin initializes a single buffer and removes the buffer when the plugin terminates.
+
+### audit_log_filter_compression
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-compression |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NONE  |
+| Valid values | NONE or GZIP |
+
+Defines the compression type for the audit log filter file. The values can be either `NONE`, the default value and no compression) or `GZIP`.
+
+
+### audit_log_filter_database
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-database |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | mysql  |
+
+Defines the audit_log_filter database, used to store the necessary tables, and is read-only. Set this option at system startup. The database name cannot exceed 64 characters or be `NULL`.
+
+Using an invalid database name prevents the use of audit log filter tables.
+
+### audit_log_filter_disable
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-disable |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Boolean  |
+| Default | OFF |
+
+Disables logging for the plugin for all connections and any sessions. 
+
+Using this variable requires the user account to have `SYSTEM_VARIABLES_ADMIN` and `AUDIT_ADMIN` privileges.
+
+### audit_log_filter_encryption
+
+| Variable Name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-encryption |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NONE  |
+| Valid values | NONE or AES |
+
+Defines the encryption type for the audit log filter file. The values can be either `NONE`, the default value and no encryption, or `AES`.
+
+### audit_log_filter_file
+
+| Variable Name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-file |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | ON  |
+
+Defines the name and suffix of the audit log filter file. The plugin writes events to this file. 
+
+The file name and suffix can be a relative path name, the plugin looks for this file in the data directory, or a full path name, the plugin uses the given value. If you use a full path name, be sure that the directory is accessible only to users who need to view the log and the server.
```

Comment:
The file name and suffix can be a relative path name, the plugin looks for this file in the data directory, or a full path name, the plugin uses the given value. -> Sounds very complicated

Reply:
be sure that - ensure